PC/svm/includes/common.h

```diff
@@ -0,0 +1,73 @@
+#ifndef COMMON
+#define COMMON
+
+#include "json.hpp"
+
+#define DATA_DIMENSIONS 2 // for Hu moments, it will be 7
+
+using json =nlohmann::json;
+
+std::vector<float> svm_build_single_element(json &element, bool debug = false)
```

Comment:
Is having implementations in .h file a good practice?

Reply:
Compilation times are longer, code is inlined (so binary is bigger). It should be moved to some .cpp but for now I will leave it.